esterni/piano-di-progetto/components/attualizzazione-dei-rischi.tex

```diff
@@ -1,5 +1,24 @@
 \documentclass[../piano-di-progetto]{subfiles}
 
 \begin{document}
+\subsection{Attualizzazione per la fase di analisi}%
+\label{sub:attualizzazione_fase_analisi}
+Nella tabella sottostante riportiamo i rischi incontrati nella fase di analisi, le modalità adottate per affrontarli e come proponiamo di gestire i suddetti rischi in futuro:
+\begin{table}[H]%
+  \label{tab:registro_delle_modifiche}
 
+  \centering
+  \rowcolors{2}{lightgray}{white!80!lightgray!100}
+
+  \begin{longtable}[H]{|p{10em}|p{17em}|p{17em}|}
+    \rowcolor{darkgray!90!}\color{white}{\textbf{Rischio}} & \color{white}{\textbf{Gestione}} & \color{white}{\textbf{Monitoraggio}} \\
+    \textbf{RTEC001} & Abbiamo incontrato problemi nell'utilizzo della strumentazione per gestire i documenti, questi problemi sono stati risolti grazie ad alcuni membri del gruppo che hanno trasmesso le loro conoscenze agli altri membri. & In futuro metteremo in cima alle priorità l'apprendimento delle nuove tecnologie delegandolo ad alcuni componenti che poi istruiranno gli altri sul funzionamento delle suddette tecnologie.\\
+    \textbf{RPER001} & Abbiamo riscontrato problemi nel gestire gli impegni accademici dei membri del gruppo, abbiamo quindi suddiviso i compiti in modo da assecondare gli impegni del gruppo. & In futuro cercheremo di comunicare il prima possibile gli impegni e di gestire il lavoro dei singoli membri in modo da evitare che questi impegni causino dei ritardi.\\
+    \textbf{RPER002} & Abbiamo riscontrato problemi nel gestire gli impegni personali dei membri del gruppo, abbiamo quindi suddiviso i compiti in modo da assecondare gli impegni del gruppo. & In futuro cercheremo di comunicare il prima possibile gli impegni e di gestire il lavoro dei singoli membri in modo da evitare che questi impegni causino dei ritardi.\\
```

Comment:
Ci sono un po' di ripetizioni di impegni : la riformulerei così. " avendo riscontrato problemi nel gestire gli impegni accademici dei membri del gruppo abbiamo deciso di suddividerci i compiti.  Prossimamente( forse bisognerebbe dare un' indicazione temporale precisa) cercheremo di pianificare e organizzare al meglio il carico di lavoro affidato ai singoli membri in modo da evitare ritardi nella consegna".

Reply:
Lo stesso problema delle ripetizioni di "impegni" va risolto anche in RPER002, visto che la parola **impegni** viene ripetuta 4 volte